src/views/manage/components/TextAreaInput.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Input } from '@datapunt/asc-ui';
+
+const TextAreaInput = ({ name, label, ...otherProps }) => {
+  return <Input as="textarea" name={name} {...otherProps} />;
```

Comment:
This might work as expect, because the input styled contain `autocomplete` and `autocorrect` attributes that might not be wanted on a textarea element.

Reply:
Not sure I understand. `autocorrect` is a valid attribute, `autocomplete` is not.